wp-content/themes/core/styles/blocks/group/dark.json

```diff
@@ -6,25 +6,51 @@
 	"blockTypes": [ "core/group" ],
 	"styles": {
 		"color": {
-			"background": "var(--wp--preset--color--base-black)",
-			"text": "var(--wp--preset--color--base-white)"
+			"background": "var(--wp--preset--color--black)",
+			"text": "var(--wp--preset--color--white)"
 		},
 		"elements": {
 			"button": {
 				"color": {
-					"background": "var(--wp--preset--color--base-white)",
-					"text": "var(--wp--preset--color--base-black)"
+					"background": "var(--wp--preset--color--white)",
+					"text": "var(--wp--preset--color--black)"
 				},
 				":hover": {
 					"color": {
-						"background": "var(--color-neutral-20)",
-						"text": "var(--wp--preset--color--base-black)"
+						"background": "var(--color-blue)",
+						"text": "var(--wp--preset--color--white)"
 					}
 				},
 				":focus": {
 					"color": {
-						"background": "var(--color-neutral-20)",
-						"text": "var(--wp--preset--color--base-black)"
+						"background": "var(--color-dark-blue)",
+						"text": "var(--wp--preset--color--white)"
+					}
+				},
+				":active": {
+					"color": {
+						"background": "var(--color-dark-blue)",
+						"text": "var(--wp--preset--color--white)"
+					}
+				}
+			},
+			"link": {
+				"color": {
+					"text": "var(--wp--preset--color--white)"
+				},
+				":hover": {
+					"color": {
+						"text": "var(--color-blue) !important"
+					}
+				},
+				":focus": {
+					"color": {
+						"text": "var(--color-blue) !important"
+					}
+				},
+				":active": {
+					"color": {
+						"text": "var(--color-dark-blue) !important"
```

Comment:
Any chance we can avoid the `!important` somehow? I hate to introduce those in the default set up.

Reply:
good call. I should have set this up to use the `--themed-*` vars in the `.is-style-dark` selector.